Visma.net/lib/VismaNetApiHelper.cs

```diff
@@ -527,11 +527,11 @@ internal static async Task<string> GetToken(string username, string password, st
             return await client.Post<VismaActionResult>(actionUrl, new object());
         }
 
-        internal static async Task<VismaActionResult> Action(VismaNetAuthorization authorization, string controller, string entityNumber, string actionName, object dto = null)
+        internal static async Task<CreateShipmentActionResult> Action(VismaNetAuthorization authorization, string controller, string entityNumber, string actionName, object dto = null)
```

Comment:
This will break other usages of Action. I thinks it's better if you just copy the function or make it generic.